packages/loaders/src/utils/useCSSSVGAnimation.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useEffect, useState } from 'react';
+
+export const useCSSSVGAnimation = () => {
```

Comment:
TL;DR; 

It injects an svg and uses translate on a `rect` to position it, it then uses the `document.elementFromPoint` method to see if the translate actually moved the element and whether css animation will work on svgs.

Reply:
Is this logic borrowed from anywhere? Might need an attribution link in the header

Reply:
Yeah good catch it' adapted from this blog post will link to it in a comment https://eprev.org/2017/01/05/how-to-detect-if-css-transforms-are-supported-on-svg/